attestation-service/verifier/src/cca/mod.rs

```diff
@@ -69,12 +86,12 @@ impl Verifier for CCA {
 
         match expected_report_data.len() {
             0..=63 => {
-                warn!("The input report_data of CCA is shorter than 64 bytes, will be padded with '\\0'.");
+                log::warn!("The input report_data of CCA is shorter than 64 bytes, will be padded with '\\0'.");
                 expected_report_data.resize(64, b'\0');
             }
             64 => {}
             _ => {
-                warn!("The input report_data of CCA is longer than 64 bytes, will be truncated to 64 bytes.");
+                log::warn!("The input report_data of CCA is longer than 64 bytes, will be truncated to 64 bytes.");
```

Comment:
We might already have import warn! macro, thus do not need this?

Reply:
I removed the import of the `warn` from log, will revert it back if you prefer to import the macro.